index.bs

```diff
@@ -217,27 +217,27 @@ interface ImageCapture {
 # PhotoCapabilities # {#photocapabilities-section}
 
 <pre class="idl">
-  [Exposed=Window] interface PhotoCapabilities {
-    readonly attribute RedEyeReduction            redEyeReduction;
-    readonly attribute MediaSettingsRange         imageHeight;
-    readonly attribute MediaSettingsRange         imageWidth;
-    readonly attribute FrozenArray&lt;FillLightMode> fillLightMode;
+  dictionary PhotoCapabilities {
+    RedEyeReduction         redEyeReduction;
+    MediaSettingsRange      imageHeight;
+    MediaSettingsRange      imageWidth;
+    sequence&lt;FillLightMode> fillLightMode;
   };
 </pre>
 
-## Attributes ## {#photocapabilities-attributes}
+## Members ## {#photocapabilities-members}
 
 <dl class="domintro">
-  <dt><dfn attribute for="PhotoCapabilities"><code>redEyeReduction</code></dfn></dt>
+  <dt><dfn dict-member for="PhotoCapabilities"><code>redEyeReduction</code></dfn></dt>
   <dd>The <a>red eye reduction</a> capacity of the source.</dd>
 
-  <dt><dfn attribute for="PhotoCapabilities"><code>imageHeight</code></dfn></dt>
+  <dt><dfn dict-member for="PhotoCapabilities"><code>imageHeight</code></dfn></dt>
   <dd>This reflects the <a>image height</a> range supported by the UA.</dd>
 
-  <dt><dfn attribute for="PhotoCapabilities"><code>imageWidth</code></dfn></dt>
+  <dt><dfn dict-member for="PhotoCapabilities"><code>imageWidth</code></dfn></dt>
   <dd>This reflects the <a>image width</a> range supported by the UA.</dd>
 
-  <dt><dfn attribute for="PhotoCapabilities"><code>fillLightMode</code></dfn></dt>
+  <dt><dfn dict-member for="PhotoCapabilities"><code>fillLightMode</code></dfn></dt>
```

Comment:
(not here but...)

We should do a `s/PhotoCapabilities object/PhotoCapabilities dictionary/` in [getPhotoCapabilities](https://w3c.github.io/mediacapture-image/#dom-imagecapture-getphotocapabilities), and/or make it match how [getPhotoSettings](https://w3c.github.io/mediacapture-image/#dom-imagecapture-getphotosettings) words it.

Reply:
I've pushed https://github.com/w3c/mediacapture-image/pull/238/commits/722398aa8e9de37dbe41bd997947c8af0e930dc9 to fix this.